src/index.js

```diff
@@ -15,10 +20,9 @@ const start = async () => {
     initSync();
     require('./web3');
     initApi();
-    startSync(true);
+    await startSync(true);
```

Comment:
is `await` here a must?

Reply:
i was playing around with getting it to handle unhandled promise exceptions, but found a diff solution for it. but either way its proper to set await on it. if it was returning then we wouldn't have to add the await.